blog/models.py

```diff
@@ -1,3 +1,12 @@
 from django.db import models
+from django.contrib.auth.models import User
+
+
+class Account(models.Model):
```

Comment:
A comment to indicate the approach this implement is using to help anyone who is still not conversant with them and also for documentation.

Reply:
Why don't you substitute a custom user, because at this point where you've used built in user, the user does not have attributes like phone_number and verification code.

Reply:
I have substituted user with custom user. For verification code I have left behind for further clarification